src/script/standard.cpp

```diff
@@ -350,39 +350,38 @@ CScript GetScriptForFreeze(CScriptNum nFreezeLockTime, const CPubKey &pubKey)
  */
 CScript GetScriptLabelPublic(const string &labelPublic)
 {
-    int64_t nLabelPublic;
-    // string hexMsg = HexStr(labelPublic);
-    nLabelPublic = int64_t(labelPublic.c_str());
-    if (nLabelPublic > 0)
+    int sizeLabelPublic = labelPublic.size();
+
+    CScript scriptDataPublic;
+
+    if (sizeLabelPublic <= 0)
+    {
```

Comment:
I think we should have a test for this if we don't already have one, if someone sends a txn with a 0 or -1 public label size.  I'm worried that something's going to break because we return an incomplete scriptDataPublic?

Reply:
nvm, I see nothing has really changed in that regard...